regression/tests/studio/test_studio_grading.py

```diff
@@ -0,0 +1,50 @@
+"""
+Test studio grading
+"""
+from bok_choy.web_app_test import WebAppTest
+from regression.pages.studio.login_studio import StudioLogin
+from regression.pages.studio.grading_studio import GradingPageExtended
+
+from regression.tests.helpers import LoginHelper, get_course_info
+
+
+class StudioGradingTest(WebAppTest):
+    """
+    Test studio grading
+    """
+    def setUp(self):
+        """
+        Initialize the page object
+        """
+        super(StudioGradingTest, self).setUp()
+        self.login_page = StudioLogin(self.browser)
+        self.course_info = get_course_info()
+        self.grading_page = GradingPageExtended(
+            self.browser, self.course_info['org'], self.course_info['number'],
+            self.course_info['run'])
```

Comment:
@raeeschachar Please refactor it according to http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#breaking-long-lines